src/notation/qml/MuseScore/NotationScene/ScorePropertiesDialog.qml

```diff
@@ -0,0 +1,220 @@
+                    Layout.preferredWidth: 175
+                    currentText: headerPart.headerTags[2]
+                    readOnly: true
+                }
+
+                StyledTextLabel {
+                    text: qsTrc("notation", "API-Level: ")
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                }
+
+                TextInputField {
+                    Layout.preferredWidth: 100
+                    currentText: headerPart.headerTags[3]
+                    readOnly: true
+                }
+
+                Item {                      //To mimic a flatbutton (so that it aligns with the open folder buttton)
+                    width: 30
+                    height: 30
+                }
+            }
+        }
+
+        SeparatorLine {}
+
+        StyledListView {
+            id: bodyTags
+            Layout.fillHeight: true
+            width: root.contentWidth
+            spacing: 5
+
+            model: scorePropertiesModel
+
+            delegate: RowLayout {
+                width: bodyTags.width
+                height: 30
+                spacing: 10
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    text: model.tagName
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                    visible: model.isStandardTag
+                }
+
+                TextInputField {
+                    Layout.preferredWidth: 110
+                    currentText: model.tagName
+                    visible: !model.isStandardTag
+                    onCurrentTextEdited: function(newTextValue) {
+                        model.tagName = newTextValue
+                    }
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    currentText: model.tagValue
+                    onCurrentTextEdited: function(newTextValue) {
+                        model.tagValue = newTextValue
+                    }
+                }
+
+                FlatButton {
+                    width: 30
+                    height: 30
+                    icon: IconCode.DELETE_TANK
+                    opacity: !model.isStandardTag
+                    onClicked: scorePropertiesModel.deleteTag(model.index)
+                }
+            }
+        }
+
+        Connections {
+            target: scorePropertiesModel
+
+            function onTagAdded(index) {
+                bodyTags.positionViewAtIndex(index, ListView.Contain)
+                bodyTags.currentIndex = index
+            }
+        }
+
+        SeparatorLine {}
+
+        RowLayout {
+            Layout.alignment: Qt.AlignBottom
+            spacing: 10
+            readonly property int buttonWidth: 132
+            height: 30
+
+            FlatButton {
+                text: qsTrc("notation", "New property")
+                width: parent.buttonWidth
```

Comment:
Is there a reason to make the width of these buttons fixed? They will have a minimum width of 132 anyway, and if they need more space in a translation, then they should be able to grow as required.

Reply:
No reason, I didn't know that :)